plugins/source-kafka/pkg/factories/command_factory.go

```diff
@@ -55,22 +55,23 @@ func (f *kafkaSourceCommandFactory) SourceCommand() *cobra.Command {
 	sourceCmd := f.defaultCommandFactory.SourceCommand()
 	sourceCmd.Use = "kafka"
 	sourceCmd.Short = "Knative eventing Kafka source plugin"
-	sourceCmd.Long = "Manage your Knative Kafka eventing sources"
+	sourceCmd.Long = "Manage Knative Kafka eventing sources"
 	return sourceCmd
 }
 
 func (f *kafkaSourceCommandFactory) CreateCommand() *cobra.Command {
 	createCmd := f.defaultCommandFactory.CreateCommand()
-	createCmd.Short = "create NAME"
+	createCmd.Use = "create NAME"
```

Comment:
this should represent all the flags which are mandatory for creating a kafka source

(check ping source usage string for example where `--sink` is a mandatory option)

Reply:
fixed.